owncloudApp/src/main/java/com/owncloud/android/ui/activity/DrawerActivity.kt

```diff
@@ -481,17 +453,16 @@ abstract class DrawerActivity : ToolbarActivity() {
      */
     protected fun setAccountInDrawer(account: Account) {
         if (drawer_layout != null) {
-            (findNavigationViewChildById(R.id.drawer_username_full) as AppCompatTextView?)?.text = account.name
+            nav_view.getHeaderView(0).drawer_username_full.text = account.name
             try {
                 val oca = OwnCloudAccount(account, this)
```

Comment:
`ownCloudAccount` instead of oca, it's not a long name

![BrilliantShadyBorer-small](https://user-images.githubusercontent.com/15261363/84390416-d9594700-abf7-11ea-8c6b-dac7a862cee0.gif)